servers/republik/graphql/resolvers/_queries/employees.js

```diff
@@ -1,14 +1,32 @@
+const { shuffle } = require('d3-array')
 const { transformUser } = require('@orbiting/backend-modules-auth')
 
-module.exports = async (_, args, {pgdb}) => {
-  const data = await pgdb.public.gsheets.findOneFieldOnly({name: 'employees'}, 'data')
+module.exports = async (_, args, { pgdb }) => {
+  const data = await pgdb.public.gsheets.findOneFieldOnly({ name: 'employees' }, 'data')
   if (!data) {
     return []
   }
-  const publishedData = data
-    .filter(d => d.published)
+  let result = data.filter(d => d.published)
 
-  const userIds = publishedData
+  if (args.shuffle) {
+    const randomOrder = shuffle(result)
+    const boosted = shuffle([
+      randomOrder.find(d => d.famous && d.gender === 'f'),
+      randomOrder.find(d => d.famous && d.gender === 'm')
+    ])
+    const rest = randomOrder.filter(d => !boosted.includes(d))
+
+    result = [
+      boosted[0],
+      rest[0],
+      boosted[1]
+    ].concat(rest.slice(1))
+      .filter(Boolean)
+      .filter((a, i, all) => all.findIndex(b => b.userId === a.userId) === i)
+      .slice(0, args.shuffle)
+  }
```

Comment:
Wouldn't it be nicer if the boosted users appear in random positions? Thus shuffling the result array again in the end?

Reply:
Current use case is to request 6 and display 3 on mobile and 6 on desktop. Generally I think it makes sense to put boosted ones in the beginning. In any display those are very visible spots.

Reply:
Alright if the sample is that small it makes perfect sense.